DUI3-DX/Converters/Rhino/Speckle.Converters.Rhino7/ToSpeckle/Raw/BrepToSpeckleConverter.cs

```diff
@@ -149,8 +155,8 @@ public SOG.Brep Convert(RG.Brep target)
           (int)trim.IsoStatus,
           (SOG.BrepTrimType)trim.TrimType,
           trim.IsReversed(),
-          trim.StartVertex.VertexIndex,
-          trim.EndVertex.VertexIndex
+          trim.StartVertex?.VertexIndex ?? -1,
```

Comment:
Why the need to default these?

Reply:
Nullablity is an issue because there was a null problem in the API.  Something was null when we didn't expect it to be

Reply:
Were you actually seeing null values?

I've just checked with Claire and the API doesn't document that this property could be null 
https://developer.rhino3d.com/api/rhinocommon/rhino.geometry.breptrim/startvertex#

Reply:
changed